src/utils/Helper.js

```diff
@@ -0,0 +1,3 @@
+export function numberWithCommas(x) {
+  return x.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ',');
```

Comment:
Moжна зробити простіше, якщо в параметрі ми отримуємо число
```suggestion
  return x.toLocaleString();
```

Reply:
дуже круто!